tests/common/beamlines/test_device_instantiation.py

```diff
@@ -21,15 +21,10 @@ def test_device_creation(RE, module_and_devices_for_beamline):
     Ensures that for every beamline all device factories are using valid args
     and creating types that conform to Bluesky protocols.
     """
-    module, devices, exceptions = module_and_devices_for_beamline
-    assert not exceptions
-    for device_name, device in devices.items():
-        assert device_name in beamline_utils.ACTIVE_DEVICES, (
-            f"No device named {device_name} was created for {module}, "
-            f"devices are {beamline_utils.ACTIVE_DEVICES.keys()}"
-        )
-        assert follows_bluesky_protocols(device)
-    assert len(beamline_utils.ACTIVE_DEVICES) == len(devices)
+    _, _, exceptions = module_and_devices_for_beamline
+    if len(exceptions) > 0:
+        raise NotConnected(exceptions)
+    # Pass test if there were no exceptions
```

Comment:
Could: This change is fine by me. You could also still check `follows_bluesky_protocols` for all returned devices but I'm not sure it gives you loads. If we're not using `follows_bluesky_protocols` now though we should remove it.